src/ui/webeiddialog.cpp

```diff
@@ -625,14 +662,15 @@ void WebEidDialog::showPinInputWarning(bool show)
 
 void WebEidDialog::resizeHeight()
 {
+    ui->langButton->move(width() - ui->langButton->width() - 20, 5);
```

Comment:
Better to make this DRY as it is used in two places, otherwise it gets error-prone when we have to update the numbers in two places:
```suggestion
    repositionLanguageButton();
```

Reply:
Removed the other location